java-ast/tests/src/test/java/org/jboss/windup/ast/java/test/JavaASTReferenceResolverTest.java

```diff
@@ -32,7 +32,7 @@ public void testHelloWorld()
                                 5, 23, 4,
                                 "public static void main(String[] argv) throws Exception {")));
         Assert.assertTrue(references.contains(
-                    new ClassReference("void", "", "void", null, ResolutionStatus.RESOLVED, TypeReferenceLocation.RETURN_TYPE, 5, 4, 108,
+                    new ClassReference("void", null, "void", null, ResolutionStatus.RESOLVED, TypeReferenceLocation.RETURN_TYPE, 5, 4, 108,
                                 "public static void main(String[] argv) throws Exception {")));
```

Comment:
I am  missing here negative checks like
```suggestion
Assert.assertTrue(references.contains(new ClassReference("testclasses.helloworld.HelloWorld.main(String[])", "testclasses.helloworld", "HelloWorld", "main",
                                ResolutionStatus.RESOLVED, TypeReferenceLocation.METHOD,
                                5, 23, 4,
                                "public static void main(String[] argv) throws Exception {")));
Assert.assertFalse(references.contains(new ClassReference("testclasses.helloworld.HelloWorld.main(String[])", "testclasses.MYEXTRAPACKAGE.helloworld", "HelloWorld", "main",
                                ResolutionStatus.RESOLVED, TypeReferenceLocation.METHOD,
                                5, 23, 4,
                                "public static void main(String[] argv) throws Exception {")));
Assert.assertFalse(references.contains(new ClassReference("testclasses.helloworld.HelloWorld.main(String[])", "testclasses.helloworld", "HelloWorldXX", "main",
                                ResolutionStatus.RESOLVED, TypeReferenceLocation.METHOD,
                                5, 23, 4,
                                "public static void main(String[] argv) throws Exception {")));                                
```

To verify that it really returns a false on different package , and the same with the class "HelloWorldX".

Reply:
Happy to add these tests.
My only concern is about merging the suggested change: it changes line 36 from
```
                                "public static void main(String[] argv) throws Exception {")));
```
to
```
Assert.assertTrue(references.contains(new ClassReference("testclasses.helloworld.HelloWorld.main(String[])", "testclasses.helloworld", "HelloWorld", "main",ResolutionStatus.RESOLVED, TypeReferenceLocation.METHOD,5, 23, 4,"public static void main(String[] argv) throws Exception {")));
```
which seems the same as [lines 29-33](https://github.com/windup/windup/blob/master/java-ast/tests/src/test/java/org/jboss/windup/ast/java/test/JavaASTReferenceResolverTest.java#L29-L33) and it would break the statement in l[ines 34-35](https://github.com/windup/windup/blob/master/java-ast/tests/src/test/java/org/jboss/windup/ast/java/test/JavaASTReferenceResolverTest.java#L34-L35).
So instead of accepting the suggested change, I'll  paste the new `Assert.assertFalse` tests in the code.
Thanks for the tests @jonathanvila :+1:

Reply:
Yep, my problem was not knowing how to specify a suggestion for several lines and not the last one.